templates/admin/src/components/theme/filter/CheckboxFilter.tsx

```diff
@@ -0,0 +1,36 @@
+import { CheckboxFilterProps } from '@lib/plume-search/filters/FilterTypes';
+import { Checkbox, FormControlLabel } from '@mui/material';
+import React from 'react';
+
+function CheckboxFilter(
+  {
+    label,
+    value,
+    disabled,
```

Comment:
```suggestion
    disabled = false,
```
pour supprimer les "?? false" plus bas